dockerfiles/mlrun/Dockerfile

```diff
@@ -32,9 +33,32 @@ WORKDIR /mlrun
 # non-recursive chmod for the run to be able to create the handler file with any security context
 RUN chmod 777 /mlrun
 
+# Install MiniConda:
+ARG MLRUN_ANACONDA_PYTHON_DISTRIBUTION="-py39"
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3${MLRUN_ANACONDA_PYTHON_DISTRIBUTION}_23.1.0-1-Linux-x86_64.sh -O ~/installconda.sh && \
+    /bin/bash ~/installconda.sh -b -p /opt/conda && \
+    rm ~/installconda.sh && \
+    /opt/conda/bin/conda clean -aqy && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
+
+ENV PATH /opt/conda/bin:$PATH
+
+# Install Open-MPI:
+ARG OMPI_VERSION=4.1.4
```

Comment:
The stable version as appears in the open-mpi site is 4.1.5 - https://www.open-mpi.org/software/ompi/v4.1/
If we're doing these changes, why not bump it while we're at it?